packages/jsii-rosetta/lib/commands/transliterate.ts

```diff
@@ -1,5 +1,6 @@
 import { Assembly, Docs, SPEC_FILE_NAME, Type, TypeKind } from '@jsii/spec';
-import { readJson, writeJson } from 'fs-extra';
+import { writeJson } from 'fs-extra';
+import { loadAssemblyFromPath } from 'jsii/lib/utils';
```

Comment:
As discussed, it is worth moving to a utilities library (separate package)... I'd rather we don't do deep imports within other packages, as I intend to eventually block them via `exports` settings.